react/features/toolbox/components/web/Toolbox.js

```diff
@@ -275,7 +275,7 @@ class Toolbox extends Component<Props, State> {
             this._shouldShowButton('videoquality') && {
                 character: 'A',
                 exec: this._onShortcutToggleVideoQuality,
-                helpDescription: 'keyboardShortcuts.videoQuality'
+                helpDescription: 'toolbar.callQuality'
```

Comment:
This doesn't seem related to the PR. Just wanted to check if you have accidentally committed it?

Reply:
My bad! Just saw it is separate commit so I guess it is not accidental :)